pyvista/core/filters/composite.py

```diff
@@ -21,21 +21,43 @@ def extract_geometry(composite):
         gf.Update()
         return wrap(gf.GetOutputDataObject(0))
 
-    def combine(composite, merge_points=False):
-        """Append all blocks into a single unstructured grid.
+    def combine(composite, merge_points=False, tolerance=0.0):
+        """Combine all blocks into a single unstructured grid.
 
         Parameters
         ----------
         merge_points : bool, optional
             Merge coincidental points.
 
+        tolerance : float
+            The tolerance to use to find coincident points when
```

Comment:
This seems to be the default, looking at https://vtk.org/doc/nightly/html/classvtkAppendFilter.html#a79f99dabfc8fa2b1a880ab0c07955226. Do we want to expose the absolute/relative choice? Somehow the "relative to mesh diagonal" seems rarely useful to me.

```suggestion
            The absolute tolerance to use to find coincident points when
```

Reply:
I don't see relative tolerance to be that helpful, so let's no expose it.